nc-news-bh-final-vite/src/components/IndividualArticle.jsx

```diff
@@ -0,0 +1,43 @@
+import {useState, useEffect} from 'react';
+import{useParams} from 'react-router-dom';
+import {fetchArticle} from '../utils/api.js';
+
+
+import ArticleFullInfo from './ArticleFullinfo';
+
+
+const IndividualArticle = ()=>{
+    const {article_id} = useParams()
+    
+
+
+    const [articleInfo, setArticleInfo] = useState([])
+    const [isLoading, setIsLoading] = useState(false)
+
+    
+
+    useEffect(()=>{
+        fetchArticle(article_id).then((data)=>{
+            const article = data.article[0]
+            setArticleInfo(article)
+            setIsLoading(false)
+        })
+
+    }, [])
+
+    
+
+    if(isLoading){
```

Comment:
This message never shows up, as you are not setting 'isLoading' to true before fetching. Please correct that.